pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -101,6 +101,9 @@ type RepositorySpec struct {
 	// The repository type.  When selected oneOf the values below should be non-nil
 	Type RepositoryType `json:"type"`
 
+	// Linting enables linting for this repository
+	Linting bool `json:"linting,omitempty"`
```

Comment:
Should this be a boolean? or a path to the lint config?  (or do we want inline configuration?)